doc/usecases/transforming_coordinate_systems.adoc

```diff
@@ -0,0 +1,31 @@
+= Transforming coordinate systems
+
+Transforming coordinates for a specific vehicle or sensor is a common task when running simulations.
+OSI uses DIN ISO 8855:2013-11 for coordinate systems and transformations between coordinate systems.
+
+//TODO: Should we add one or more sentences about the mathematical operations involved?
```

Comment:
@max-rosin, are they mentioned elsewhere, if yes we should add a link to them?
If no then, yes we should populate this section with more data.

Reply:
No, this was more of an idea I had while reading the topic. I was especially thinking about the order of rotations.

Reply:
CCB: Should be tackled after migration is done.